src/tools/cephfs/JournalTool.cc

```diff
@@ -62,7 +62,7 @@ void JournalTool::usage()
     << "    <output>: [summary|list|binary|json] [--path <path>]\n"
     << "\n"
     << "General options:\n"
-    << "  --rank=filesystem:mds-rank|all Journal rank (mandatory)\n"
+    << "  --rank=filesystem:{mds-rank|all} Journal rank or all (mandatory)\n"
```

Comment:
```suggestion
    << "  --rank=filesystem:{mds-rank|all} Journal rank or "all" (mandatory)\n"
```

Perhaps adding quotes will make it non-ambiguous for users that they can pass the string "all".

Reply:
I think mds-rank|all the |all part makes it clear that you can pass all.